AllReadyApp/Web-App/AllReady.UnitTest/Controllers/TaskApiControllerTests.cs

```diff
@@ -77,7 +77,7 @@ public class TaskApiControllerTests
             var mediator = new Mock<IMediator>();
             mediator.Setup(x => x.SendAsync(It.IsAny<EventByEventIdQuery>())).ReturnsAsync(new Event());
             mediator.SetupSequence(x => x.SendAsync(It.IsAny<TaskByTaskIdQuery>()))
-                .ReturnsAsync(new AllReadyTask()).ReturnsAsync(null);
+                .ReturnsAsync(allReadyTask).ReturnsAsync(null);
```

Comment:
I had some issues initially. @mgmccarthy upgraded to a new version of MOQ to support the SetupSequence and for some reason VS was not restoring it for me. I ended up rebooting and then doing a restore before it finally got caught up and compiled.

If that fails, you might want to try a dotnet restore from the cmd line.